components/dropdown/dropdown-content-mixin.js

```diff
@@ -264,6 +264,13 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 		this.opened = true;
 	}
 
+	async requestResize() {
```

Comment:
Can we rename this to `resize` so its consistent with the public `resize` method in dialog? https://github.com/BrightspaceUI/core/tree/master/components/dialog

Reply:
Done